src/core/qgsrange.h

```diff
@@ -167,6 +167,19 @@ class QgsRange
       return false;
     }
 
+    bool operator==( const QgsRange<T> &other ) const
+    {
+      return mLower == other.mLower &&
```

Comment:
not sure if it is double do you want to use double comparison?

Reply:
Do you mean qgsDoubleNear?

Reply:
y

Reply:
Done